### ISSUE TITLE
Override Costume Bugs

### ISSUE THREAD

**Issue by XanoPhobious** [nipkownix/re4_tweaks#278]
### Validation

- [X] Game has been updated to the latest RE4HD release (v1.0.1: https://www.re4hd.com/?p=9471)
- [X] Data has been [validated with SFV](https://www.re4hd.com/?p=9454) (a guide to using QuickSFV is available at that link, use that with this 1.0.1 SFV: [BIO4-HDProject1.0.1-SFV.zip](https://github.com/nipkownix/re4_tweaks/files/8057899/BIO4-HDProject1.0.1-SFV.zip))

### Describe your issue here (drag+drop ZIP to attach it)

Sorry if my English is wierd. My game version is 1.0.6 on Win64 English. There are few bugs regarding "Override Costume" that I encounter:
1. If I choose RPD (Leon) and Armor (Ashley), Leon costume will glitch.
![rpd](https://user-images.githubusercontent.com/97661288/179658204-ef389e34-4b9a-4582-a083-f58c325bebdf.jpg)

2. Ada's costume won't change regardless what I choose for her.
3. Ada's costume will change depend on what I choose for Ashley. If I pick Popstar, Ada's costume will be RE2 (short skirt?). If I choose Normal/Armor for Ashley, Ada's costume will be Normal (long skirt?).
4. If choose Normal for Ada (from ingame; the F1 button), the dinput8.ini will have duplicate entries right after "AdaCostume = Normal". If I choose it by editing the dinput8.ini file, Ada's costume will be empty (when I view it from the F1 button)
![ada](https://user-images.githubusercontent.com/97661288/179658317-4c4b04c5-48ca-4338-be82-dc2752453363.jpg)
 and the log file [DINPUT8.log](https://github.com/nipkownix/re4_tweaks/files/9137169/DINPUT8.log) will have entries from dinput8.ini

**Comment by nipkownix** [comment]
Yeah, I'm aware of the cutscene issues, but there isn't much I can do about those.
Basically, if I fix Leon, [Ashley gets messed up.](https://github.com/nipkownix/re4_tweaks/issues/191#issuecomment-1155860531)
Maybe I can take another look at this in the future, though.

**Comment by XanoPhobious** [comment]
I see, so the cutscene issue is known. Ok, no problem. I hope one day you can find what causing that.
Cutscene; I has not playing as Ada. Is the Override Costume for Ada only for when playing as Ada?

**Comment by Mister-Curious** [comment]
It might be worth checking out how the Companion DLL handles this task by allowing users to use a folder for overriding cutscene models. I can lend any of my expertise on how this system works if anyone is interested.

**Comment by ddubs13** [comment]
Choosing Jacket for Leon causes a crash when getting beheaded by Salvador after 1-1. I have yet to find out what other deaths cause a crash.
[bio4.exe.20221029164420.zip](https://github.com/nipkownix/re4_tweaks/files/9894656/bio4.exe.20221029164420.zip)

**Comment by emoose** [comment]
Just noticed that `PlClothSetLeon` includes some scenario flag check inside it (680729 in 1.1.0 checks for `SCF_R106_EVENT` flag), and seems to only init some cloth stuff if that flag isn't set, wonder if that could be causing some of the costume override issues... @nipkownix do you know if we patch any of the PlCloth funcs for the overrides?

**Comment by Grammernatzi** [comment]
Choosing Armor Ashley causes the game to crash when you're decapitated as any non-Mafia Leon costume. This is endlessly repeatable.
